### PR TITLE
Added Emoji8 source code to Samples folder

### PR DESCRIPTION
**Pull request by mccoykillian**

### CODE REVIEW

File: Samples/Emoji8/UWP/cs/Emoji8/Services/IntelligenceService.cs [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+            try
+            {
+                _model = await LearningModel.LoadFromStorageFileAsync(modelStorageFile);
+            }
+            catch(Exception ex)
+            {
+                Debug.WriteLine(ex.Message);
+            }
+
+            _device = new LearningModelDevice(LearningModelDeviceKind.Default);
```
Comment (martinb35) [inline Samples/Emoji8/UWP/cs/Emoji8/Services/IntelligenceService.cs]: if you're just going to use the default device, you actually don't need to specify it - you can just create a session without the device parameter: "_session = new LearningModelSession(_model);".
alternatively, you could try to run this on the GPU too if the user has one.

File: Samples/Emoji8/UWP/cs/Emoji8/Pages/EmotionPage.xaml.cs [hunk 2]
```diff
@@ -0,0 +1,310 @@
...
+    /// <summary>
+    /// An empty page that can be used on its own or navigated to within a Frame.
+    /// </summary>
+    public sealed partial class EmotionPage : Page
+    {
+        private static List<Emoji> _resultsEmojis { get; set; }
+        private DispatcherTimer EmojiTimer;
+        private int countdownNum;
+        private int numEmojisLeftToPerform = Constants.EMOJI_NUMBER_TO_DISPLAY;
+        private double nextPercentage;
```
Comment (youngkim93) [inline Samples/Emoji8/UWP/cs/Emoji8/Pages/EmotionPage.xaml.cs]: Can we have a default value for this for consistency?

Comment (youngkim93) [reply Samples/Emoji8/UWP/cs/Emoji8/Pages/EmotionPage.xaml.cs]: same for countdownNum :)

File: Samples/Emoji8/UWP/cs/Emoji8/Pages/EmotionPage.xaml.cs [hunk 3]
```diff
@@ -0,0 +1,310 @@
...
+            }
+            else
+            {
+                UpdateCountdownRing();
+            }
+
+            if (countdownNum % Constants.DIVISIBLE_FACTOR == 0)
+            {
+                //display pattern 5...4...3...2...1...5... [...] ...2...1...0
+                if ((numEmojisLeftToPerform > -1 && countdownNum != 0) ||
```
Comment (youngkim93) [inline Samples/Emoji8/UWP/cs/Emoji8/Pages/EmotionPage.xaml.cs]: I haven't read through all the details of this, but could numEmojisLeftToPerform be negative? By looking at the variable name it seems to me like it should never be negative :)

**Pull request merged**